File: src/targets/javascript/jquery/fixtures/multipart-file.js
const form = new FormData();
<<<<<<< HEAD
form.append('foo', 'src/fixtures/files/hello.txt');
=======
form.append('foo', 'test/fixtures/files/hello.txt');
>>>>>>> upstream/master

const settings = {
  async: true,
  crossDomain: true,
<<<<<<< HEAD
  url: 'https://httpbin.org/anything',
=======
  url: 'http://mockbin.com/har',
>>>>>>> upstream/master
  method: 'POST',
  headers: {},
  processData: false,
  contentType: false,
  mimeType: 'multipart/form-data',
  data: form
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: src/targets/python/requests/fixtures/multipart-file.py
import requests

<<<<<<< HEAD
url = "https://httpbin.org/anything"

files = {"foo": ("src/fixtures/files/hello.txt", open("src/fixtures/files/hello.txt", "rb"), "text/plain")}

response = requests.post(url, files=files)

print(response.text)
=======
url = "http://mockbin.com/har"

files = { "foo": "open('test/fixtures/files/hello.txt', 'rb')" }

response = requests.post(url, files=files)

print(response.json())
>>>>>>> upstream/master
